### ISSUE TITLE
Code Autocompletion of Tailwind CSS IntelliSense Not Functioning

### ISSUE THREAD

**Issue by ikore908** [getcursor/cursor#1165]
1) screenshot
It doesn't function with the cursor.
![image](https://github.com/getcursor/cursor/assets/30616878/5e1edae4-df16-4c0c-8579-fa911a293040)

It is functioning in VSCode.
![image](https://github.com/getcursor/cursor/assets/30616878/e36b5fef-466b-4384-aad9-e802bca42e0f)

2) operating system
macOS Sonoma 14.2.1

3) steps

1. Opening a tsx file with the cursor.
2. Write Tailwind syntax in the className.

**Comment by elvstejd** [comment]
Here's how I solved it:

### Identifying the error
1. I hit `Ctrl+ Shift + P` to open Command Palette.
2. Wrote "Focus on output view".
3. On the section that opens, selected "Tailwind CSS IntelliSense" to see its logs.
4. It was indeed crashing. From what I could see, it was trying to read some scss file, which is not normally supported by the extension.
a. I wasn't even using scss.
b. But I guess some dependency of my project was.

### Solving the error
1. Hit `Ctrl+ Shift + P` again to open Command Palette.
2. Looked for "Open User Settings JSON"
3. Pasted this in the configuration:
```json
"tailwindCSS.files.exclude": [
        "**/.git/**",
        "**/node_modules/**",
        "**/.hg/**",
        "**/.svn/**",
        "**/*.scss",
    ],
```
- Basically told it to ignore scss files, among other folders it should never go into, just in case. 
4.  `Ctrl+ Shift + P` > "Reload window"
5. Working again!

**Comment by ikore908** [comment]
@elvstejd 
Thanks a lot for your help!

**Comment by jordisantamaria** [comment]
It's still not working on me, tailwindcss Intellisense didn't crush. I'm not getting classNames autocompletion in cursor but it works in VScode.
